packages/react-strict-dom/src/dom/runtime.js

```diff
@@ -71,7 +71,7 @@ const styles = stylex.create({
     borderStyle: 'solid'
   },
   strong: {
-    fontWeight: 'bolder'
+    fontWeight: 'bold'
```

Comment:
Interesting. In the past the `strong` element was `bolder` by default in UA style sheets, as the HTML spec expects https://html.spec.whatwg.org/#phrasing-content-3

But it seems that some time over the last few years, browsers starting using `bold` - this is the case today in Chrome and Safari.